cli/server/infrastructure/fileStore.ts

```diff
@@ -522,8 +533,12 @@ export class FileStore {
 				let mappings: Mapping[] = (yaml.load(content) as Mapping[]) || [];
 
 				const originalLength = mappings.length;
+				// we need to re-calculate the hash
 				mappings = mappings.filter((m) => {
-					return `${m.control_id}:${m.uuid}` !== compositeKey;
+					console.log('Delete Mapping: ' + JSON.stringify(m));
+					m.hash = createHash('sha256').update(JSON.stringify(m)).digest('hex');
+					console.log('Delete  Hash: ' + m.hash);
+					return `${m.control_id}:${m.hash}` !== compositeKey;
```

Comment:
This is probably fine, but I think we could alternatively save the hash as a separate var (rather than on the map) and be able to skip the "cleaning" later?

```suggestion
					const hash = createHash('sha256').update(JSON.stringify(m)).digest('hex');
					return `${m.control_id}:${hash}` !== compositeKey;
```

Reply:
oh yes, that is much cleaner. Adding that in the code and deleting the hash cleanup from the mappings.